src/ScottPlot/Renderable/Legend.cs

```diff
@@ -14,6 +14,21 @@ namespace ScottPlot.Renderable
 {
     public class Legend : IRenderable
     {
+        /// <summary>
```

Comment:
I don't think this is actually a change due to this PR

To make these lines (and changelog above) not appear as part of this PR on GitHub you can _update from master_ then this PR may look a little cleaner with fewer files/lines modified

Reply:
> update from master
This may fix the failing `Test_Legend_RendersWithoutRenderingPlotFirst()` test too